jre/guide/java/java_jre_deployment_config_configured/java_jre_deployment_config_properties/bash/jre.sh

```diff
@@ -2,8 +2,8 @@
 JAVA_CONFIG="/etc/.java/deployment/deployment.config"
 JAVA_PROPERTIES="/etc/.java/deployment/deployment.properties"
 
-grep -q "^deployment.system.config=file://${JAVA_CONFIG}$" ${JAVA_CONFIG} && \
-sed -i "s;deployment.system.config=.*;deployment.system.config=file:\/\/${JAVA_PROPERTIES};g" ${JAVA_CONFIG}
-if ! [ $? -eq 0 ] ; then
-  echo "deployment.system.config=file://${JAVA_PROPERTIES}" >> ${JAVA_CONFIG}
-fi
+sed -i "/deployment.system.config=.*/d" ${JAVA_CONFIG}
```

Comment:
What are we making a change when we don't need to make a change? This seems like a regression and doesn't lead itself to idempotent behavior.

Reply:
The previous change defined the property incorrectly. If a system is upgrading from the previous release, they would end up with two entries, one being correct and one not. The suggested change removes any previous entry and applies the correct entry.

We have to test both cases of a new install and upgrading from the previous release.